src/lighteval/models/transformers/transformers_model.py

```diff
@@ -249,13 +273,21 @@ def __init__(
             logger.info(f"Using Data Parallelism, putting model on device {self._device}")
             self.model = self.model.to(self._device)
         if config.compile:
-            logger.info("Compiling the model")
-            self.model.model.compile()
+            try:
+                logger.info("Compiling the model")
+                self.model.model.compile()
+            except AttributeError as e:
+                logger.warn("Could not compile the model because: ", e)
```

Comment:
```suggestion
                logger.warning("Could not compile the model because: ", e)
```

https://docs.python.org/3/library/logging.html#logging.Logger.warning